app/assets/javascripts/homeroom_table/HomeroomTable.js

```diff
@@ -4,34 +4,27 @@ import SortHelpers from '../helpers/sort_helpers.jsx';
 import Cookies from 'js-cookie';
```

Comment:
The goal here is to also renames app and test files to look like what @kevinrobinson did in #1224.